features/keychain/api/index.js

```diff
@@ -1,10 +1,14 @@
 const createKeychainApi = ({ keychain }) => {
-  const exportKey = async ({ seedId, keyId, exportPrivate }) =>
-    keychain.exportKey({ seedId, keyId, exportPrivate })
-
   return {
     keychain: {
-      exportKey,
+      exportKey: (...args) => keychain.exportKey(...args),
+      sodium: {
+        signDetached: keychain.sodium.signDetached,
+        getKeysFromSeed: (...args) =>
+          keychain.sodium
+            .getSodiumKeysFromSeed(...args)
+            .then(({ secret, ...publicKeys }) => publicKeys),
```

Comment:
seems a bit safer, in case we add anything private there later
```suggestion
            .then(({ box, sign }) => ({ box, sign })),
```